twitter.html

```diff
@@ -0,0 +1,484 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>twitter</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <!-- the navegation part is a bit different as i did exactly like the real twitter page,
+  please check the pics inserted in the folder -->
+  <body>
+    <div class="layout">
+      <div class="search">
+        <div class="search-button">
+          <img src="search.png" alt="search-icon" />
+          <input type="text" placeholder=" Search Twitter" />
+        </div>
+      </div>
+      <header>
+        <h2><span class="home1">Home</span></h2>
+        <div class="effect">
+          <svg viewbox="0 0 24 24" aria-hidden="true" class="star">
```

Comment:
It's a nice practice to use the `<img>` tag when using SVGs as it reduces the amount of lines in our html file and makes it more readable.